### ISSUE TITLE
Forward doesn't work for inlined fields

### ISSUE THREAD

**Issue by r0x** [yourlabs/django-autocomplete-light#664]
Hi there,
I was using dal  with the forward option and I noticed (as pointed out in #639) that it doesn't work as well as the live demo of the feature at [http://dal-yourlabs.rhcloud.com/admin/linked_data/testmodel/add/]:
if a value is selected for the owner (for inlined elements) the values proposed by the widget of the "test" field don't change.
I discovered that in select2.js, the script tries to match the DOM element with name="$('[name=' + forward[key] + ']')" but when the field is inlined it doesn't have the name of the model field. Instead it has a name like inlinedModelName-x-x where x is a digit from 0 to 9.
I have made a little fix for select2.js  as follows :

(from line 26 of select2.js)
```
var forward = element.attr('data-autocomplete-light-forward');
if (forward !== undefined) {
   var data_forward = {};
   forward = forward.split(',');
   prefix = $(this).attr('name').match(/\w+-(\d+-)+/);
   if( prefix != null)
   {
      prefix = prefix[0];
   }
   else
   {
      prefix = '';
   }
   for (var key in forward) {
      data_forward[forward[key]] = $('[name=' + prefix + forward[key] + ']').val();
   }

   data.forward = JSON.stringify(data_forward);
}
```

Hope it helps!

-----------------
**EDIT**
Sorry, I just noticed that the bug was fixed 4 days ago with the commit a8f89ab.

**Comment by jpic** [comment]
Upgrade version.

**Comment by jpic** [comment]
Sorry, something between travis and openshift broke and this affected continuous deployment, the demo website doesn't have this commit from 5 days ago. Thanks for reporting !

**Comment by jpic** [comment]
Fixed by 00a110f 

I guess I had done a password reset or something on openshift itself xD
